tests/draft-next/propertyDependencies.json

```diff
@@ -0,0 +1,115 @@
+[
+    {
+        "description": "propertyDependencies doesn't act on non-objects",
+        "schema": {
+            "propertyDependencies": {
+                "foo": {"bar": false}
+            }
+        },
+        "tests": [
+            {
+                "description": "ignores booleans",
+                "data": true,
+                "valid": true
+            },
+            {
+                "description": "ignores integers",
+                "data": 123,
+                "valid": true
+            },
+            {
+                "description": "ignores floats",
+                "data": 1.0,
+                "valid": true
+            },
+            {
+                "description": "ignores strings",
+                "data": "abc",
+                "valid": true
+            },
+            {
+                "description": "ignores arrays",
+                "data": [],
+                "valid": true
+            },
+            {
+                "description": "ignores null",
+                "data": null,
+                "valid": true
+            }
+        ]
+    },
+    {
+        "description": "propertyDependencies doesn't act on non-string property values",
+        "schema": {
+            "propertyDependencies": {
+                "foo": {"bar": false}
+            }
+        },
+        "tests": [
+            {
+                "description": "ignores booleans",
+                "data": {"foo": false},
+                "valid": true
+            },
+            {
+                "description": "ignores integers",
+                "data": {"foo": 2},
+                "valid": true
+            },
+            {
+                "description": "ignores floats",
+                "data": {"foo": 1.1},
+                "valid": true
+            },
+            {
+                "description": "ignores objects",
+                "data": {"foo": {}},
+                "valid": true
+            },
+            {
+                "description": "ignores objects wth a key of the expected value",
+                "data": {"foo": {"bar": "baz"}},
+                "valid": true
+            },
+            {
+                "description": "ignores objects with the expected value nested in structure",
+                "data": {"foo": {"baz": "bar"}},
+                "valid": true
+            },
+            {
+                "description": "ignores arrays",
+                "data": {"foo": []},
+                "valid": true
+            },
+            {
+                "description": "ignores null",
+                "data": {"foo": null},
+                "valid": true
+            }
+        ]
+    },
+    {
+        "description": "disallowing values",
+        "schema": {
+            "propertyDependencies": {
+                "foo": {"bar": {"minProperties": 2}}
```

Comment:
This is good but sorry I didn't mean to remove the boolean ones, usually we have both, e.g.: https://github.com/json-schema-org/JSON-Schema-Test-Suite/blob/4cfead7ba7b1af840eaf95f19a088c89ed67b5a9/tests/draft2019-09/patternProperties.json#L120-L155

Typically boolean schemas require special casing so it's good to basically mirror those tests I think, and keep these.

Reply:
okay.  I'll see if I can work this up.